### DISCUSSION TITLE
오브젝트에 닿였을시 Scene 씬 전환

### DISCUSSION THREAD

**Discussion by HyunBin3** [naverz/zepeto-studio-kor#1184] [5. Q&A]
혼자 독학 하면서 제페토 공부하고 있는데
OnTriggerEnter를 사용해서 오브젝트랑 닿았을 시 씬을 넘어가게 하고싶은데
계속 오류가 뜨네요 혹시 방법이 있을가요?

**Comment by jefflim69** [comment]
> 혼자 독학 하면서 제페토 공부하고 있는데 OnTriggerEnter를 사용해서 오브젝트랑 닿았을 시 씬을 넘어가게 하고싶은데 계속 오류가 뜨네요 혹시 방법이 있을가요?

Build Setting 에 Scene 들이 모두 포함 되어 있으신가요 ?

**Reply by HyunBin3** [reply]
넵 빌드는 해놓은 상태 입니다

**Reply by jefflim69** [reply]
> 넵 빌드는 해놓은 상태 입니다

대체로 발생할 수 있는 요인이
1. 오브젝트의 Collider 에 isTrigger 가 Checking 되어 있지 않거나 ,
2. Build Setting 에 Load 해야 할 Scene 이 추가 되어 있지 않을 경우 입니다 .
혹 , 관련 부분의 Capture 를 올려 주시면 도움을 좀 더 드릴 수 있을 것 같네요 .
